expected-errs.txt

```diff
@@ -2,42 +2,42 @@ WARNING: Multiple elements have the same ID 'dom-decodeerrorcallback-error'.
 WARNING: Multiple elements have the same ID 'dom-decodesuccesscallback-decodeddata'.
 Deduping, but this ID may not be stable across revisions.
-LINE 1910: Can't find the 'contextOptions' argument of method 'OfflineAudioContext/OfflineAudioContext(numberOfChannels, length, sampleRate)' in the argumentdef block.
-LINE 3019: Can't find the 'destinationNode' argument of method 'AudioNode/connect(destinationParam, output)' in the argumentdef block.
-LINE 3019: Can't find the 'input' argument of method 'AudioNode/connect(destinationParam, output)' in the argumentdef block.
-LINE 3120: Can't find the 'destinationNode' argument of method 'AudioNode/disconnect(destinationParam, output)' in the argumentdef block.
-LINE 3133: Can't find the 'destinationNode' argument of method 'AudioNode/disconnect(destinationParam, output)' in the argumentdef block.
-LINE 3148: Can't find the 'destinationNode' argument of method 'AudioNode/disconnect(destinationParam, output)' in the argumentdef block.
-LINE 3148: Can't find the 'input' argument of method 'AudioNode/disconnect(destinationParam, output)' in the argumentdef block.
-LINE 10077: Can't find method '['AudioWorkletProcessor/process(inputs, outputs, parameters))']'.
-LINE 1407: No 'method' refs found for 'AudioContext()'.
-LINE 1910: No 'method' refs found for 'OfflineAudioContext(contextOptions)'.
-LINE 1934: No 'method' refs found for 'OfflineAudioContext(numberOfChannels, length, sampleRate)'.
-LINE 2291: No 'method' refs found for 'AudioBuffer()'.
-LINE 4158: No 'method' refs found for 'AnalyserNode()'.
-LINE 4618: No 'method' refs found for 'AudioBufferSourceNode()'.
-LINE 5853: No 'method' refs found for 'BiquadFilterNode()'.
-LINE 6286: No 'method' refs found for 'ChannelMergerNode()'.
-LINE 6398: No 'method' refs found for 'ChannelSplitterNode()'.
-LINE 6480: No 'method' refs found for 'ConstantSourceNode()'.
-LINE 6588: No 'method' refs found for 'ConvolverNode()'.
-LINE 6844: No 'method' refs found for 'DelayNode()'.
-LINE 7009: No 'method' refs found for 'DynamicsCompressorNode()'.
-LINE 7451: No 'method' refs found for 'GainNode()'.
-LINE 7558: No 'method' refs found for 'IIRFilterNode()'.
-LINE 7731: No 'method' refs found for 'MediaElementAudioSourceNode()'.
-LINE 7837: No 'method' refs found for 'MediaStreamAudioDestinationNode()'.
-LINE 7929: No 'method' refs found for 'MediaStreamAudioSourceNode()'.
-LINE 8005: No 'method' refs found for 'MediaStreamTrackAudioSourceNode()'.
-LINE 8144: No 'method' refs found for 'OscillatorNode()'.
-LINE 8533: No 'method' refs found for 'PannerNode()'.
-LINE 8934: No 'method' refs found for 'PeriodicWave()'.
-LINE 9246: No 'method' refs found for 'StereoPannerNode()'.
-LINE 9375: No 'method' refs found for 'WaveShaperNode()'.
-LINE 9806: No 'method' refs found for 'AudioWorkletNode()'.
+LINE: Can't find the 'contextOptions' argument of method 'OfflineAudioContext/constructor(numberOfChannels, length, sampleRate)' in the argumentdef block.
```

Comment:
Why did the line numbers disappear?  Seems like a possible regression in bikeshed.  Doesn't block this PR though.

Reply:
I think this is intended and I did make a mistake last time I regenerated the file `expected-errs.txt`. It's better to not include line numbers here so that we don't have to regenerate the file when the line number change.